lms/djangoapps/instructor_task/subtasks.py

```diff
@@ -494,7 +494,6 @@ def update_subtask_status(entry_id, current_task_id, new_subtask_status, retry_c
         _release_subtask_lock(current_task_id)
 
 
-@transaction.commit_manually
```

Comment:
Why not just use the `@transaction.atomic` decorator here and avoid the reformatting below?